### ISSUE TITLE
Issue: Not a directory when trying to cache autoprefixer.js

### ISSUE THREAD

**Issue by ptrikutam** [ai/autoprefixer-rails#105]
Thanks so much for your hard work on this gem! We're trying to use this for one of our projects and we're running into the following issue when setting up autoprefixer.

We've installed the gem and Rails boots up properly, but when compiling the CSS we get a runtime error:
```
Errno::ENOTDIR - Not a directory @ rb_sysopen - path-to-rvm-gem-directory/autoprefixer-rails-6.4.0.1/lib/autoprefixer-rails/processor.rb/../../../vendor/autoprefixer.js
  (in path-to-app/app/assets/stylesheets/project-chimp/main.scss):
  autoprefixer-rails (6.4.0.1) lib/autoprefixer-rails/processor.rb:118:in `read_js'
  autoprefixer-rails (6.4.0.1) lib/autoprefixer-rails/processor.rb:123:in `build_js'
  autoprefixer-rails (6.4.0.1) lib/autoprefixer-rails/processor.rb:112:in `runtime'
  autoprefixer-rails (6.4.0.1) lib/autoprefixer-rails/processor.rb:28:in `process'
  autoprefixer-rails (6.4.0.1) lib/autoprefixer-rails/sprockets.rb:20:in `run'
  autoprefixer-rails (6.4.0.1) lib/autoprefixer-rails/sprockets.rb:59:in `render'
  sprockets (2.11.0) lib/sprockets/context.rb:197:in `block in evaluate'
  sprockets (2.11.0) lib/sprockets/context.rb:194:in `evaluate'
```

I'm on rails 4.1.13 &  sprockets 2.11.0.

I'm having this issue on my local machine (haven't tried it on production yet). I was able to get autoprefixer to play nicely when I changed the line in `processor.rb` like so: https://github.com/somawater/autoprefixer-rails/commit/688ff894c85e785d88f4301fb55e7db675dff215

Any idea what's going on here? I'm a bit stumped.

**Comment by ai** [comment]
So, you fix it with that line?

Send me a PR (but I will be on laptop only after 20 hours)

**Comment by ptrikutam** [comment]
Sure thing, will do right now.

**Comment by ptrikutam** [comment]
Submitted the PR -- I'm not quite sure if the PR is what we want, so please double-check and make sure it's not going to break anything :)

**Comment by ai** [comment]
Released in 6.4.0.2
